### PR TITLE
Update references to the packages directory.

### LINKED ISSUE #144: Update docs when packages directory is no longer created by default
According to https://github.com/dart-lang/sdk/issues/27399, `--packages-dir` will become false in 1.20. We need to update the docs accordingly when 1.20 is stable.

### PR DESCRIPTION
**Pull request by kwalrath**
Staged at: https://kw-www-dartlang-1.firebaseapp.com

@nex3 would you be able to review this change?

Closes #144.

### CODE REVIEW

File: src/tools/faq.md [hunk 1]
```diff
@@ -45,11 +45,9 @@ hoping to address the root cause but it will take a while to get there.
 
 #### Q. What are all the "packages" directories for?
```
Comment (nex3) [inline src/tools/faq.md]: If we're removing references to the `packages/` directory elsewhere, I think we can safely ditch this question as well. It won't be "frequently asked" anymore.

File: src/tools/pub/package-layout.md [hunk 2]
```diff
@@ -69,16 +69,15 @@ enchilada/
     Leave it out of source control unless your package is an
     [application package](/tools/pub/glossary#application-package).
 
-\*** The `packages` directories are
-     created when you run `pub get`,
-     unless you specify the flag `--no-packages-dir`.
+\*** The `packages` directories are created
```
Comment (nex3) [inline src/tools/pub/package-layout.md]: I think we can probably remove this bullet point too. I don't expect any users to pass `--packages-dir` who aren't already familiar enough with Dart development not to need this note.

File: src/_guides/libraries/create-library-packages.md [hunk 3]
```diff
@@ -205,7 +205,7 @@ or any `packages` directories. For more information, see
 
 {% comment %}
 PENDING: here just to make it easy to find discussions of `packages`...
-{% include coming-release.html %}
+{% include packages-dir.html %}
```
Comment (nex3) [inline src/_guides/libraries/create-library-packages.md]: What does this refer to?

Comment (kwalrath) [inline src/_guides/libraries/create-library-packages.md]: The actual packages-dir.html include (when not commented out) produces a standard note about packages/.packages. E.g. see https://kw-www-dartlang-1.firebaseapp.com/guides/libraries/private-files#packages-and-packages.

Comment (kwalrath) [inline src/_guides/libraries/create-library-packages.md]: The following files refer to packages-dir.html:

src/_guides/libraries/create-library-packages.md
src/_guides/libraries/private-files.md
src/_tutorials/libraries/shared-pkgs.md
src/tools/faq.md
src/tools/pub/cmd/pub-downgrade.md
src/tools/pub/cmd/pub-get.md
src/tools/pub/cmd/pub-upgrade.md
src/tools/pub/get-started.md
src/tools/pub/glossary.md
src/tools/pub/package-layout.md
src/tools/pub/publishing.md

**Comment by kwalrath** [comment]
Updated and re-staged.

**Pull request merged**